app/subcommands/repo/list

```diff
@@ -3,6 +3,8 @@
 # vim: ft=sh ts=4 sw=4 sts=4 noet
 set -euf
 
+filter="${1:-all}"
```

Comment:
now that this subcommand take an optional parameter it needs a `# Usage` comment at the start to go with the description. This should follow the usual argument conventions where [] indicates an optional parameter and <> wraps its name and indicates its position.